edx-platform/pearson-bsg-theme/lms/templates/footer.html

```diff
@@ -10,155 +10,68 @@
-              ## You can either replace this link entirely or update
-              ## the FOOTER_ORGANIZATION_IMAGE in Django settings.
-              ## If you customize FOOTER_ORGANIZATION_IMAGE, then the image
-              ## can be included in the footer on other sites
-              ## (e.g. a blog or marketing front-end) to provide a consistent
-              ## user experience.  See the branding app for details.
-              <img alt="${_('organization logo')}" src="${footer['logo_image']}">
-            </a>
-          </p>
-        </div>
+      <nav class="nav-legal" aria-label="${_('Legal')}">
+        <ul>
+          % for item_num, link in enumerate(footer['legal_links'], start=1):
+            % if link['name'] == 'honor_code':
+              <% continue %>
+            % endif
+            <li class="nav-legal-0${item_num}">
+              <a href="${link['url']}">${link['title']}</a>
+            </li>
+          % endfor
+          <li><a href="${footer['edx_org_link']['url']}">${footer['edx_org_link']['text']}</a></li>
+        </ul>
+      </nav>
+    </div>
 
-        ## Site operators: Please do not remove this paragraph! This attributes back to edX and makes your acknowledgement of edX's trademarks clear.
-          <p class="copyright">${footer['copyright']}
-              % if icp_license_info.get('icp_license'):
-                ${u" | {text}".format(text=icp_license_info.get('text'))}
-                <a href="${icp_license_info.get('icp_license_link', '#')}">
-                    ${u" {icp}".format(icp=icp_license_info.get('icp_license'))}
-                </a>
-              % endif
-          </p>
-
-        <nav class="nav-legal" aria-label="${_('Legal')}">
-          <ul>
-            % for item_num, link in enumerate(footer['legal_links'], start=1):
-              <li class="nav-legal-0${item_num}">
-                <a href="${link['url']}">${link['title']}</a>
-              </li>
-            % endfor
-            <li><a href="${footer['edx_org_link']['url']}">${footer['edx_org_link']['text']}</a></li>
-          </ul>
-        </nav>
-      </div>
+    ## Please leave this link and use one of the logos provided
+    ## The OpenEdX link may be hidden when this view is served
+    ## through an API to partner sites (such as marketing sites or blogs),
+    ## which are not technically powered by OpenEdX.
+    % if not hide_openedx_link:
+    <div class="footer-about-openedx">
+      ## Site operators: Please do not remove this paragraph! This attributes back to edX and makes your acknowledgement of edX's trademarks clear.
+      <p class="copyright">${footer['copyright']}
+        % if icp_license_info.get('icp_license'):
+        ${u" | {text}".format(text=icp_license_info.get('text'))}
+        <a href="${icp_license_info.get('icp_license_link', '#')}">
+          ${u" {icp}".format(icp=icp_license_info.get('icp_license'))}
+        </a>
+        % endif
+      </p>
```

Comment:
Is this position change necessary? @amalbas

Reply:
yes, if not the copyright will be on the side of legal menu which is not as the production site

Reply:
Got it.